### PR TITLE
Routing and sample data

### PR DESCRIPTION
**Pull request by paterasMSFT**
Adds routing and the ability to add sample data. Also fetches data from the "objectives" collection, just as a proof of concept.

### CODE REVIEW

File: src/test.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "Id": 444,
+    "Name": "Obj1",
+    "Version": "2019-01-01 2pm",
+    "Owner": ["SBorg"],
+    "Area": ["Wit X"],
+    "Target": "2019-07-31",
+    "Progress": 0.2,
+    "KRs": [
+        123,
```
Comment (Nianwens) [inline src/test.json]: Do we have KR sample data somewhere?

File: src/SampleDataPage.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
+import * as React from "react";
+import { ExtensionDataService } from "VSS/SDK/Services/ExtensionData";
+
+interface ISampleDataPageState {
+    collection: string,
```
Comment (Nianwens) [inline src/SampleDataPage.tsx]: Rename it to more specific, objective and KRs?

File: src/test.json (hunk 1 above)
Comment (paterasMSFT) [reply src/test.json]: Not yet. Those are fake.

File: src/SampleDataPage.tsx (hunk 2 above)
Comment (paterasMSFT) [reply src/SampleDataPage.tsx]: The user inputs the collection name. So if you want to add a KR, you'd put KR (or whatever) in that field when inserting data.

**Pull request merged**